### PR TITLE
fixed path for macOS without XCode installed

### PR DESCRIPTION
**Pull request by u3dreal**

### CODE REVIEW

File: deps/clew/src/clew.cpp [hunk 1]
```diff
@@ -164,7 +164,7 @@ int clewInit()
 #ifdef _WIN32
     const char *paths[] = {"OpenCL.dll", NULL};
 #elif defined(__APPLE__)
-    const char *paths[] = {"/Library/Frameworks/OpenCL.framework/OpenCL", NULL};
+    const char *paths[] = {"/System/Library/Frameworks/OpenCL.framework/OpenCL", NULL};
```
Comment (Dade916) [inline deps/clew/src/clew.cpp:167]: This patch is a bad idea, it should be:

```
const char *paths[] = {"/Library/Frameworks/OpenCL.framework/OpenCL", 
 "/System/Library/Frameworks/OpenCL.framework/OpenCL", NULL};
```

If you want, add a paths to the list but don't remove them.

Comment (u3dreal) [reply deps/clew/src/clew.cpp:167]: /Library/Frameworks/OpenCL does not even exist !!!

/System/Library/Frameworks/OpenCL  is what we need for standard.

/Library/frameworks is where the user ( installed ) Frameworks go.. Cuda etc

OpenCL is a System Framework. ... as far as i know

Comment (u3dreal) [reply deps/clew/src/clew.cpp:167]: I think if Xcode is installed the sdk Frameworks are used ... if not the one in /System/Library/Framewoks

Comment (Dade916) [reply deps/clew/src/clew.cpp:167]: "/Library/Frameworks/OpenCL.framework/OpenCL" is the path used by Blender so it should be valid (in some case).

Comment (u3dreal) [reply deps/clew/src/clew.cpp:167]: Hmm OpenCL is disabled on OSX  for long. So maybe that is deprecated..
on Catalina OpenCL is also in /System/Library 
Luxcore won't run on everything less that 10.13 
But if you want we just add it ..

Comment (u3dreal) [reply deps/clew/src/clew.cpp:167]: I added it 
[https://github.com/LuxCoreRender/LuxCore/pull/388](url)

Comment (u3dreal) [reply deps/clew/src/clew.cpp:167]: I added it back
[https://github.com/LuxCoreRender/LuxCore/pull/388](url)

Comment (maguilar72G) [reply deps/clew/src/clew.cpp:167]: Umm, this broken link!

**Pull request merged**